### PR TITLE
Add feature request issue template

### PR DESCRIPTION
**Pull request by lornasong**
Preview:

Screen Shot 2020-10-13 at 5 05 38 PM

### CODE REVIEW

**Comment by lornasong** [comment]
I looked at the feature request template across our 6 major products to come up with the proposed template. One additional section that I chose not to include but open to feedback: “Is your feature request related to a problem? Please describe”. I also saw some variations in wording and took some liberties and combined some language

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<!--Please search the existing issues for relevant feature & enhancement requests and use the 👍 reaction to add upvotes to pre-existing requests.-->
+
+#### Description
+A clear and concise overview of the feature / enhancement.
+
+#### Use Cases
+Any relevant use-cases that would help us understand the use / need / value. Please share them as they can help us decide on acceptance and prioritization.
+
+#### Alternative Solutions
+A clear and concise description of any alternative solutions or features you've considered.
```
Comment (eikenb) [inline .github/ISSUE_TEMPLATE/feature_request.md:15]: Maybe add 'workarounds'... "any alternative solutions, workarounds or features you've considered"

**Review by eikenb — APPROVED** [review]
My one comment/suggestion is not a blocker. Include it if you like or not.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (lornasong) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Yea, I think it makes sense - we want users to include workaround-level information. Added!

**Pull request merged**